components/identities/pfpGallery.tsx

```
@@ -56,7 +56,7 @@ const PfpGallery: FunctionComponent<PfpGalleryProps> = ({
             <div className="flex flex-col align-middle items-center">
               <h2 className={styles.title}>NO NFTS FOUND</h2>
               <p className={styles.subtitle}>
-                You don&apos;t own any whitelisted NFTs yet. Get your first NFT
+                You don't own any whitelisted NFTs yet. Get your first NFT
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)